packages/reassure-compare/src/utils/validate.ts

```diff
@@ -6,6 +6,7 @@ import { hasDuplicateValues } from './array';
 export const performanceMetadataSchema = z.object({
   branch: z.string().optional(),
   commitHash: z.string().optional(),
+  creationDate: z.string().optional(),
```

Comment:
Pls use `z.string().datetime().optional()` to validate also the format of the date string (https://github.com/colinhacks/zod/releases/tag/v3.20)